src/main/scala/engine/SimulationHistory.scala

```diff
@@ -2,18 +2,23 @@ package engine
 
 import model.genome.KindsUtils
 import model.mutation.Mutation
+import model.world.Environment.Factor
 import model.world.Generation.Population
-import util.PimpScala.RichTuple2
-import model.world.Reproduction.{ generateInitialCouple, nextGenerationBunnies }
-import model.world._
+import model.world.Reproduction.{generateInitialCouple, nextGenerationBunnies}
+import model.world.{Climate, Environment, Generation, Summer}
 
 import scala.language.implicitConversions
 
-object SimulationHistory {
+object SimulationHistory{
 
   type History = List[Generation]
 
-  var history: History = List(Generation(Environment(Summer(), List.empty), generateInitialCouple.toSeq))
```

Comment:
Quindi andava mantenuto il toSeq aggiunto in pimp scala, dato che generateInitialCouple  dovrebbe ritornare una Couple e non una Population nella versione refactored